site/gamedata.php

```diff
@@ -123,33 +118,34 @@
         </div>
     </section>
     <section>
-        <div class="row mb-5">
+        <div class="row mb-3">
             <div class="col">
-                <h3 class="mb-0">Player Activity</h3>
-                <?php echo '<strong><span id="player-activity-date">' . $activity_text . '</span></strong>' ?>
+                <h3 class="mb-0">Monthly Player Activity</h3>
             </div>
-            <?php if (isset($game_files)) : ?>
-                <div class="col text-end">
-                    <nav class="text-nowrap">
-                        <?php echo '<button id="month-prev" type="button" class="btn btn-outline-secondary" ' . $prev_disabled . '><i class="bi bi-chevron-left"></i> ' . $prev_month . '</button>'; ?>
-                        <?php echo '<button id="month-next" type="button" class="btn btn-outline-secondary" ' . $next_disabled . '>' . $next_month . ' <i class="bi bi-chevron-right"></i></button>'; ?>
-                    </nav>
-                </div>
-            <?php endif; ?>
         </div>
         <!-- Chart -->
         <?php require 'includes/chart.php'; ?>
     </section>
+    
+    <div class="row mb-5 gy-2 ms-1">
+        <div class="<?php echo (isset($game_files) ? 'col' : 'me-1 col'); ?>">
+            <div class="bg-primary rounded p-2 row" id="stats" data-year="<?php echo $selected_year; ?>" data-month="<?php echo $selected_month; ?>">
+                <div class="col" id="stats-header"><?php echo htmlspecialchars($month_name . ' ' . $selected_year); ?></div>
+                <div class="col text-end" id="num-plays">No Plays</div>
+            </div>
+        </div>
+        <?php if (isset($game_files)) : ?>
+        <div class="month-nav-wrapper col-md-2 col-sm-4 gy-2 text-end me-5 me-lg-0">
```

Comment:
I think `col-md-3` fits the content better than `col-md-2`. With the change the breakpoint for the width override in the `month-nav-wrapper` class, you should also be able to remove the margins - `me-5 me-lg-0`